src/Shared/Microsoft.DotNet.Scaffolding.Helpers/General/ProjectModelHelper.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System.Reflection;
+
+namespace Microsoft.DotNet.Scaffolding.Helpers.General;
+
+internal static class ProjectModelHelper
+{
+    internal static Dictionary<string, string> ShortTfmDictionary = new Dictionary<string, string>()
```

Comment:
Read only? Also looks like this is only used with a .ToList call afterwards (at least in Helpers) so maybe it should be a list not a dictionary? Though perhaps that'll make merging Shared/Helpers later harder?

Reply:
will do this as part of the Helper - Shared rework.